stardata/Form1.cs

```diff
@@ -0,0 +1,235 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.Data;
+using System.Drawing;
+using System.IO;
+using System.Linq;
+using System.Net.Mime;
+using System.Text;
+using System.Threading.Tasks;
+using System.Windows.Forms;
+using static System.Net.Mime.MediaTypeNames;
+
+namespace stardata

```

Comment:
consider separation the concerns by organizing functionality into separate classes. for example creating and using 3 following classes StarDataEntry, StarDataParser, and StarDataManager. Where The StarDataEntry class would represent a single entry of star data, with properties to store the various attributes of a star.
The StarDataParser class would provide a static method Parse() to parse a string input into a StarDataEntry object.
And the StarDataManager class would handle the saving and loading of star data, as well as providing formatted output in different formats.

That would promote code reusability by encapsulating star data parsing, saving, loading, and formatting functionalities into reusable classes, with each class having a clear purpose and well-defined methods. This would make it easier to understand and maintain the codebase and allow these functionalities to be used in different parts of the application or even in other applications.